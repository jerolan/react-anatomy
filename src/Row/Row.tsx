import styled from 'styled-components';

import FlexBox from '../FlexBox';
import { GridRowColScalesProps } from '../types';

const Row = styled(FlexBox)<GridRowColScalesProps>`
  flex-direction: row;
  flex: ${props => props.xs || 1};

  @media (min-width: 768px) {
    flex: ${props => props.sm};
  }

  @media (min-width: 992px) {
    flex: ${props => props.md};
  }

  @media (min-width: 1200px) {
    flex: ${props => props.lg};
  }
`;

export default Row;
