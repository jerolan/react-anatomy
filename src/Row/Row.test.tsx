import React from 'react';
import renderer from 'react-test-renderer';

import Row from './Row';

describe('<Row />', () => {
  it('renderenders without crashing', () => {
    const component = renderer.create(<Row />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
