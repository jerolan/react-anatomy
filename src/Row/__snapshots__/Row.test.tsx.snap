// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> renderenders without crashing 1`] = `
<div
  className="sc-bdVaJa sc-bwzfXH lfgcWI"
/>
`;
